### ISSUE TITLE
filterscripts - documentation (Trac #785)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#779]
I have documentation for L2 2011/2012 here:
http://icecube.wisc.edu/~dschultz/docs/L2/index.html

I'll also probably finish out the series for other years.

This should probably live some place official, like the actual L2 project(s).  But that doesn't get built into our documentation because they aren't in icerec/trunk.

Is there another good place to put this?

Thoughts?

Migrated from https://code.icecube.wisc.edu/ticket/785
```json
{
    "status": "closed", 
    "changetime": "2016-01-22T20:59:42", 
    "description": "I have documentation for L2 2011/2012 here:\nhttp://icecube.wisc.edu/~dschultz/docs/L2/index.html\n\nI'll also probably finish out the series for other years.\n\nThis should probably live some place official, like the actual L2 project(s).  But that doesn't get built into our documentation because they aren't in icerec/trunk.\n\nIs there another good place to put this?\n\nThoughts?", 
    "reporter": "david.schultz", 
    "cc": "nega, olivas, claudio.kopper", 
    "resolution": "fixed", 
    "_ts": "1453496382708350", 
    "component": "combo reconstruction", 
    "summary": "filterscripts - documentation", 
    "priority": "major", 
    "keywords": "docs, L2", 
    "time": "2014-10-16T19:05:09", 
    "milestone": "Long-Term Future", 
    "owner": "david.schultz", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150519T21:32:49: 
* david.schultz changed milestone from "" to "offline V15-07-00"
* david.schultz changed status from "new" to "accepted"
* david.schultz commented:

> add a milestone so we are more pressured to fix this.

**Comment by icecube-trac** [comment]
Trac update at 20150810T20:31:17: 
* olivas commented:

> Milestone offline V15-07-00 deleted


* olivas changed milestone from "offline V15-07-00" to ""

**Comment by icecube-trac** [comment]
Trac update at 20151209T16:46:25: david.schultz changed milestone from "" to "Long-Term Future"

**Comment by icecube-trac** [comment]
Trac update at 20160122T20:59:42: 
* david.schultz changed status from "accepted" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 141184:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141184"
> move L2 documentation to a proper home. tidy up main docs. closes #779
> ```
